src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +15,32 @@ public MovieDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
```

Comment:
Recommendation - you can use try with resources here

Reply:
OK, but there is the recommendation from "Common mistakes" - [Use try with finally, where you use transaction and try with resources with Read operations](https://mate-academy.github.io/jv-program-common-mistakes/hibernate/relations/relations_checklist)